src/pages/en/index.astro

```diff
@@ -1,12 +1,384 @@
+                <VisionCard
+                    data-aos="fade-up"
+                    thumbnail={Audience}
+                    text={t("home.vision.1")}
+                />
+                <VisionCard
+                    data-aos="fade-up"
+                    data-aos-delay="200"
+                    thumbnail={StairImg}
+                    text={t("home.vision.2")}
+                />
+                <VisionCard
+                    data-aos="fade-up"
+                    data-aos-delay="400"
+                    thumbnail={StageImg}
+                    text={t("home.vision.3")}
+                />
+            </div>
+        </Container>
+        <div class="relative">
+            <Container>
+                <div
+                    data-aos="fade-up"
+                    data-aos-duration="1000"
+                    class="flex flex-col-reverse justify-between gap-10 md:flex-row"
+                >
+                    <Section
+                        tag={t("home.tedx.tag")}
+                        description={t("home.tedx.description")}
+                    >
+                        <div slot="headline">
+                            What is <span class="c-underline">TEDx?</span>
+                        </div>
+                        <Button
+                            href="https://www.ted.com/about/programs-initiatives/tedx-program"
+                            target="_blank"
+                        >
+                            {t("home.tedx.more")} &nbsp;
+                            <Icon name="mdi:open-in-new" />
+                        </Button>
+                    </Section>
+                    <img
+                        src={TedxImg.src}
+                        alt="TEDx"
+                        class="aspect-video md:aspect-auto md:w-1/2 object-cover border border-gray-500 rounded-2xl duration-200 shadow-[0_0_2rem_0_rgba(0,0,0,0.3)] hover:shadow-primary-600"
+                    />
+                </div>
+            </Container>
+            <img
+                src={TedxDecoreImg.src}
+                alt=""
+                class="absolute top-0 -translate-y-2/3 md:w-full -z-10 left-1/2 md:left-0 -translate-x-1/2 max-w-screen-2xl md:max-w-full md:translate-x-0"
+            />
+        </div>
+
+        <div class="relative">
+            <Container>
+                <Section
+                    tag={t("home.events.tag")}
+                    description={t("home.events.description")}
+                    style="md:text-center mb-16"
+                >
+                    <div slot="headline">
+                        A <span class="c-underline">Legacy</span> of Inspiration
+                    </div>
+                </Section>
+                <div
+                    data-aos="fade-up"
+                    data-aos-delay="200"
+                    class="swiper-event flex gap-2"
+                >
+                    <div class="swiper-wrapper">
```

Comment:
Maybe make the `swiper-wrapper` div or EventCard center in mobile view. Because we have only one past event

![image](https://github.com/user-attachments/assets/66e2e88d-454e-4322-ac47-9ca16cc30494)

Reply:
OK